README.md

```diff
@@ -40,3 +42,38 @@ This will do the following:
 - Install root dependencies
 - Install all package dependencies using `lerna bootstrap`
 - Initialize git hooks (`npx @escape.tech/mookme init --only-hook --skip-types-selection`)
+
+### Running A Client
+
+Environments:
+
+- *:dev runs against local services
+- *:stage runs against the staging testnet
+- *:prod runs against production infrastructure
+
+```bash
+# web
+npm run web:dev
+npm run web:stage
+npm run web:prod
+
+# desktop
+npm run desktop:dev
+npm run desktop:stage
+npm run desktop:prod
+
+# mobile (append -- --device to target a physical device)
```

Comment:
Just tried this and it didn't default to my device. Only worked when I put `-- --device "my iphone"`. The `npm run ios:device` command defaulted to the phone if connected